test/verify/check-storage-stratis

```diff
@@ -617,6 +617,64 @@ class TestStorageStratis(storagelib.StorageCase):
         # And the pool should be full again
         b.wait_visible("button:contains(Create new filesystem):disabled")
 
+    @testlib.skipImage("Stratis too old", "rhel-8-*", "centos-8-*")
+    def testPoolResize(self):
+        m = self.machine
+        b = self.browser
+
+        self.login_and_go("/storage")
+
+        dev = "/dev/sda"
+        m.add_disk("4G", serial="DISK1")
+        b.wait_in_text("#drives", dev)
+
+        # Create a logical volume that we will later grow
+        m.execute(f"vgcreate vgroup0 {dev}; lvcreate vgroup0 -n lvol0 -L 1500000256b")
+        b.wait_in_text("#devices", "vgroup0")
+
+        # Create a pool
+        self.dialog_with_retry(trigger=lambda: self.devices_dropdown("Create Stratis pool"),
+                               expect=lambda: self.dialog_is_present('disks', "lvol0"),
+                               values={"disks": {"lvol0": True}})
+        b.wait_in_text("#devices", "pool0")
+        b.wait_in_text("#devices", "1.50 GB Stratis pool")
+
+        # Grow the logical volume in Cockpit, the pool should grow automatically
+        b.click('.sidepanel-row:contains("vgroup0")')
+        b.wait_visible('#storage-detail')
+        self.content_tab_action(1, 1, "Grow")
+        self.dialog({"size": 1600})
+        b.go("#/")
+        b.wait_in_text("#devices", "1.60 GB Stratis pool")
+
+        # Grow the logical volume from outside of Cockpit, the pool should complain
+        m.execute("lvresize vgroup0/lvol0 -L +100000256b")
+        b.wait_visible('.sidepanel-row:contains(pool0) .ct-icon-exclamation-triangle')
+        b.click('.sidepanel-row:contains("pool0")')
+        b.wait_visible('#storage-detail')
+        b.wait_visible('.pf-v5-c-alert:contains("This pool does not use all the space")')
```

Comment:
Would be nice to train ourselves to move to `wait_in_text()` for such things now.

Reply:
I guess we want to stop using `:contains`? Unfortunately, `wait_in_text` only works when there is a single `.pf-v5-c-alert` on the page, or when we give the alert an `id` or something else unique....

Reply:
Not important here, we need a more global fix for dropping sizzle anyway. Resolving.